### ISSUE TITLE
Remove extra space in help screen with no abstract

### ISSUE THREAD

**Issue by erica** [apple/swift-argument-parser#156]
`OVERVIEW: 

Check the time at a given location.

- For example: "now Sao Paolo" (diacriticals), "denver" (case), "20202"
  (postal codes), "Lincoln Memorial" (well known placemark) etc.
- Set a reference time with `at`, for example "now --at 5:45PM Bath UK".
- Valid times include: 5PM, 5P, 5:30PM, 5:30P, 17:30. (No spaces.)`

The material in the command summary uses a lot of excess white space, which fills up the vertical screen. Many users stick to 80x24 terminal geometries and won't necessarily use `more` or `less` to page output. Squeezing the vertical information by optionally eliminating the extra line after OVERVIEW, etc, would create more succinct output and prevent the user from having to search so much for the `USAGE` line.

**Comment by natecook1000** [comment]
It looks like you've specified a discussion, but not an abstract. Shouldn't be a problem to eliminate that. 👍 

```swift
struct Example: ParsableCommand {
    static let configuration = CommandConfiguration(discussion: "This is an example command.")
}
```
Currently outputs:
```
$ swift run example
OVERVIEW: 

This is an example command.

USAGE: example

OPTIONS:
  -h, --help              Show help information.
```
Should output:
```
$ swift run example
OVERVIEW: This is an example command.

USAGE: example

OPTIONS:
  -h, --help              Show help information.
```

**Comment by natecook1000** [comment]
Fixed by #161.
